source/extensions/common/wasm/wasm.cc

```diff
@@ -231,6 +231,9 @@ void Wasm::getFunctions() {
 #define _GET(_fn) wasm_vm_->getFunction("envoy_" #_fn, &_fn##_);
   _GET(on_resolve_dns)
 #undef _GET
+#define _GET(_fn) wasm_vm_->getFunction("envoy_" #_fn, &_fn##_);
```

Comment:
You can just include it in the previous define/undef block instead of duplicating the defines.

Reply:
fixed, good catch